Content.Server/Zombies/ZombieSystem.cs

```diff
@@ -43,6 +45,9 @@ public sealed partial class ZombieSystem : SharedZombieSystem
         [Dependency] private readonly SharedPopupSystem _popup = default!;
         [Dependency] private readonly SharedRoleSystem _role = default!;
 
+        [DataField]
+        public ProtoId<NpcFactionPrototype> Faction = "Zombie";
```

Comment:
```suggestion
        public readonly ProtoId<NpcFactionPrototype> Faction = "Zombie";
```
make it readonly so it cannot be modified from elsewhere